Zend/zend_compile.h

```diff
@@ -360,6 +364,10 @@ struct _zend_op_array {
 	uint32_t required_num_args;
 	zend_arg_info *arg_info;
 	/* END of common elements */
+    struct {
+        zend_uchar type; // the accessor type (getter or setter)
+        uint32_t property_offset;
+    } accessor;
```

Comment:
It's not great to extend all op_arrays with rarely used fields for getters only.